src/components/RoomNameInput.js

```diff
@@ -26,6 +26,12 @@ const propTypes = {
     /** ID of policy whose room names we should be checking for duplicates */
     policyID: PropTypes.string,
 
+    /** Whether to show the error on change of input or not */
+    shouldShowErrorOnChange: PropTypes.bool,
+
+    /** Error text to show when shouldShowErrorOnChange is false */
```

Comment:
It's just error text. We shouldn't refer to any of the component logic in our description of this prop.

```suggestion
    /** Error text to show */
```